### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
- Updates strings 
- Adds/updates FAQs
- Increases hero slide iteration time
- Updates beta invite email
- Adds plaintext version of beta invite email

### CODE REVIEW

File: privaterelay/templates/faq.html [hunk 1]
```diff
@@ -27,14 +27,42 @@ <h2 class="faq-headline">
           What about spam?
         </h2>
         <p class="faq-answer">
-          We use Amazon SES as our Email Service Provider, which enforces industry standard spam controls. We also have the ability to enforce even more strict spam filtering.      </p>
+          While Relay does not filter for spam, our email partner Amazon SES does block spam and malware. If Relay forwards messages you don’t want, you can update your Relay settings to block messages from the alias forwarding them. If you see a broader problem of unwanted email from all of your aliases, please report this to us so we can consider adjusting the SES spam thresholds for this service.
```
Comment (groovecoder) [inline privaterelay/templates/faq.html:30]: We could make "report this to us" go to our GitHub issues? Or maybe the AMO page for reviews?

File: privaterelay/templates/faq.html [hunk 2]
```diff
@@ -27,14 +27,42 @@ <h2 class="faq-headline">
...
+        <p class="faq-answer">
+          If you report these as spam, your email provider will see Relay as the source of spam, not the original sender.
+        </p>
+      </section>
+      <section class="faq">
+        <h2 class="faq-headline">
+          Can I reply to messages using my Relay alias?
+        </h2>
+        <p class="faq-answer">
+          Relay does not yet offer the ability to reply using an alias. If you try, nothing will happen. We are working on additional features to let you <a href="https://github.com/mozilla/fx-private-relay/issues/151" rel="noopener noreferrer" class="text-link">send an automated response</a> or <a href="https://github.com/mozilla/fx-private-relay/issues/99" rel="noopener noreferrer" class="text-link">reply anonymously to senders</a>.
```
Comment (groovecoder) [inline privaterelay/templates/faq.html:41]: Hmm ... https://github.com/mozilla/fx-private-relay/issues/151 is not for the *users* to send an automated response. It's for *us* to send an automated response reminding users that they can't yet reply to relay emails.

File: privaterelay/templates/faq.html [hunk 3]
```diff
@@ -27,14 +27,42 @@ <h2 class="faq-headline">
...
+        </p>
+      </section>
+      <section class="faq">
+        <h2 class="faq-headline">
+          It seems like a site will not accept my Relay address or is blocking addresses using @relay.firefox.com. Why is this happening? What can I do about it?
+        </h2>
+        <p class="faq-answer">
+          Some sites may not accept an email address that includes a subdomain (ie, the “relay” portion of @relay.firefox.com) and others have stopped accepting all addresses except those from Gmail, Hotmail, or Yahoo accounts. As Firefox Relay grows in popularity and issues more aliases, our service might be placed on a blocklist. If you are not able to use a Relay alias, please let us know
```
Comment (groovecoder) [inline privaterelay/templates/faq.html:49]: "let us know" needs a period. and the link could go to our GitHub issues or the AMO page for reviews.

File: privaterelay/templates/faq.html [hunk 4]
```diff
@@ -27,14 +27,42 @@ <h2 class="faq-headline">
...
+        </p>
+      </section>
+      <section class="faq">
+        <h2 class="faq-headline">
+          Can I make up my own relay address using the @relay.firefox.com domain?
+        </h2>
+        <p class="faq-answer">
+           Not currently, but we are considering new features including letting you create your own addresses with a designated domain.
```
Comment (groovecoder) [inline privaterelay/templates/faq.html:57]: We could link this to issue https://github.com/mozilla/fx-private-relay/issues/55 if we want people to comment there and/or upvote.

File: privaterelay/templates/faq.html (hunk 2 above)
Comment (lesleyjanenorton) [reply privaterelay/templates/faq.html:41]: drop the link?

File: privaterelay/templates/faq.html (hunk 1 above)
Comment (lesleyjanenorton) [reply privaterelay/templates/faq.html:30]: Your call -  I think GitHub may make more sense, but AMO might be easier for people unfamiliar with GitHub

File: privaterelay/templates/faq.html (hunk 2 above)
Comment (groovecoder) [reply privaterelay/templates/faq.html:41]: Yeah, I'd drop the link for sure. Could drop that mention completely too? If you think Michelle would be fine with that.

File: privaterelay/templates/faq.html (hunk 1 above)
Comment (groovecoder) [reply privaterelay/templates/faq.html:30]: Let's do AMO. Monitor users aren't likely to know GitHub well enough to file issues.

**Pull request merged**